contracts/Controller.sol

```diff
@@ -738,13 +738,12 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         require(_checkVaultId(_args.owner, _args.vaultId), "Controller: invalid vault id");
 
         MarginVault.Vault memory vault = getVault(_args.owner, _args.vaultId);
+        bool hasShorts = _isNotEmpty(vault.shortOtokens);
+        bool hasLongs = _isNotEmpty(vault.longOtokens);
```

Comment:
I used the term `hasShort` and `hasLong` instead of shorts and longs in MarginCalculator. I'm fine with either but better make it consistent!

Reply:
Cool will change it.